requirements.txt

```
@@ -1,4 +1,4 @@
-git+https://github.com/edx/xblock-utils.git@v1.0.2#egg=xblock-utils==1.0.2
+git+https://github.com/edx/xblock-utils.git@v1.2.2#egg=xblock-utils==1.2.2
```

I recommended this since this block is available on PyPI now, and that makes for a much quicker install. 
```suggestion
xblock-utils==1.2.2
```